### DISCUSSION TITLE
Move Windows 11 taskbar to sides or top of screen?

### DISCUSSION THREAD

**Discussion by randolf** [Open-Shell/Open-Shell-Menu#1909] [Q&A]
Can Open Shell be used to move the Windows 11 taskbar to the sides or the top of the screen?  I know many people who want to do this, and it seems that Windows 11 has removed access to this feature, so I'm hoping Open Shell can come to the rescue here.

Many users have wide screens and find it more convenient to move their task bar to the left-hand or right-hand side of the screen so they can see more tasks at once.

Some users also prefer to move their task bar to the top of the screen.  This is particularly useful when remotely controlling Windows computers that have larger screens when we don't want to use screen scaling options, because then we don't need to pan the screen down to access the various functions that are provided by the task bar.

Thanks!

**Comment by Ibuprophen** [comment]
There was a time (in very early releases of Win 11) that this could be done, but it appears that this ability was removed via Microsoft Developers. 

The following is a good source/link that provides more clarity regarding this. 

https://answers.microsoft.com/en-us/windows/forum/all/how-do-i-move-the-taskbar-to-the-top-of-the-screen/d105291a-43dd-45bc-abe3-0c929c8ff6a7

I'm not (personally) aware of this ability using something via a 3rd party. Maybe someone else here has any working ideas/suggestions for this. 

Good Luck! :-)) 

~Ibuprophen

**Reply by randolf** [reply]
Thanks for explaining this ... not-so-wonderful situation. It's disappointing that this user-friendly feature that many people like and use was removed from the OS.

**Reply by Ibuprophen** [reply]
This is one example of why I only use Win 10 & 11 on in a spare PC for testing and such. I still use Win 8.1 on my primary PC. 

~Ibuprophen

**Reply by Celltronixpak** [reply]
https://windhawk.net/mods/taskbar-on-top use this tool

**Reply by MyloshB** [reply]
FYI, Windhawk is a good app, but it contains one problem from my point of view...
Windhawk changes the user's color of the Taskbar (defined in Open-Shell) to the default black :-/
So, I have also tried the ExplorerPatcher app and it is a bit better, IMHO :-)
But I think, the possibility of having the Taskbar on the top of the screen should be a nice Xmas gift from Open-Shell programmers...

**Comment by karenmcd** [comment]
Apparently there's an app 'explorerpatcher' that allows to move the start menu to the top now? https://www.elevenforum.com/t/bring-back-the-ability-to-move-taskbar-to-top-and-sides.28459/ https://github.com/valinet/ExplorerPatcher

I've been using openshell for so long, i'm wondering if this functionality can be brought to openshell to work on windows 11 or if it will be just easier to install this explorerpatcher when i am forced to switch to w11?

it seems so stupid that all our menus and icons to interact with saving, creating changing things, as well as browser tabs and everything are at the top of the screen and they force us to move the mouse all the way to the bottom of the screen, ruining workflows and adding so much extra literal work to access a menu we need all the time.

I use linux for most of my stuff anyway, and gnome hot corner and taskbars/menus thankfully still can be put up top, but for me what a dealbreaker this was for windows 11

Thanks for all the great work on open shell over the years. I love it so much!
